src/golang/lib/repos/tests/random.go

```diff
@@ -16,3 +21,22 @@ func randString(n int) string {
 	}
 	return string(b)
 }
+
+// pollIDs polls IDs n times.
```

Comment:
The meaning of poll is to check the status of (usually repeatedly) so I'm not too sure `pollIDs` and `pollUserIDs` are the most intuitive names for these functions.